code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/dao/impl/ServiceExposeMapDaoImpl.java

```diff
@@ -197,7 +197,7 @@ public ServiceExposeMap getServiceIpExposeMap(Service service, String ipAddress)
     }
 
     @Override
-    public List<? extends Service> getGlobalServices(long accountId) {
+    public List<? extends Service> getActiveServices(long accountId) {
```

Comment:
This is a bit embarrassing.  I had an idea over the weekend, but just half-way executed and then got distracted.  Anyways, this call wasn't retrieving global services (I was originally tempted to just do a string regex in the DB field to do the filtering there); but, that's too hacky.